app/src/main/java/com/asfoundation/wallet/ui/iab/payments/carrier/confirm/CarrierFeeFragment.kt

```diff
@@ -0,0 +1,169 @@
+package com.asfoundation.wallet.ui.iab.payments.carrier.confirm
+
+import android.content.Context
+import android.content.res.Configuration
+import android.graphics.drawable.Drawable
+import android.os.Bundle
+import android.text.SpannedString
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.text.buildSpannedString
+import androidx.core.text.color
+import com.asf.wallet.R
+import com.asfoundation.wallet.GlideApp
+import com.asfoundation.wallet.ui.iab.IabView
+import com.asfoundation.wallet.util.CurrencyFormatUtils
+import com.asfoundation.wallet.util.WalletCurrency
+import com.asfoundation.wallet.util.getStringSpanned
+import com.jakewharton.rxbinding2.view.RxView
+import dagger.android.support.DaggerFragment
+import io.reactivex.Observable
+import kotlinx.android.synthetic.main.dialog_buy_buttons_payment_methods.*
+import kotlinx.android.synthetic.main.fragment_carrier_confirm.*
+import java.math.BigDecimal
+import java.util.*
+import javax.inject.Inject
+
+class CarrierFeeFragment : DaggerFragment(), CarrierFeeView {
+
+  private val formatter = CurrencyFormatUtils()
```

Comment:
I don't have a strong opinion eitherway, but I think it is fine on fragment since it is a format utils. Formatting is generally done on the view side AFAIK.